### PR TITLE
Wdp181202-18

### PR DESCRIPTION
**Pull request by Ma-Der**
Problem: New Furniture should be a slider.
Solution: Add Tiny-Slider to New Furniture section.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -178,8 +179,9 @@ <h3>New furniture</h3>
           </div>
         </div>
       </div>
-      <div class="row">
-        <div class="col-xl-3 col-lg-4 col-md-6 col-sm-12">
+
+      <div class="my-slider">
+        <div>
```
Comment (xkxd) [inline src/index.html]: I'd suggest adding some class to these elements to avoid confusion

File: src/sass/components/_footer.scss [hunk 2]
```diff
@@ -80,3 +82,30 @@ footer {
...
+    .bottom-bar {
+      position: relative;
+      .unidentify-object {
+        position: absolute;
+        bottom: 90%;
+        left: 0;
+        width: 100%;
+      }
+      .copyright {
+        p {
```
Comment (xkxd) [inline src/sass/components/_footer.scss]: again, it's not necessary to have so complicated selector here, as you'd easily achieve the same on root level with
```
.copyright  {
    p {
        text-align...
    }
}
```

File: src/scripts/app.js [hunk 3]
```diff
@@ -1 +1,27 @@
-console.log('hello');
\ No newline at end of file
+console.log('hello');
```
Comment (xkxd) [inline src/scripts/app.js]: please remove it

File: src/scripts/app.js [hunk 4]
```diff
@@ -1 +1,25 @@
-console.log('hello');
\ No newline at end of file
+var slider = tns({
+    container: '.my-slider',
```
Comment (pawel-schmidt) [inline src/scripts/app.js]: 4 spaces indentation while from line 9 you have 2 spaces.

File: src/sass/components/_product-box.scss [hunk 5]
```diff
@@ -103,3 +103,46 @@
...
+        
+        &:hover {
+          background-color: $primary;
+        }
+      }
+    }
+    .tns-nav-active {
+      background-color: $primary;
+    }
+}
```
Comment (pawel-schmidt) [inline src/sass/components/_product-box.scss]: Reindent, please.

**Pull request merged**